src/app/pages/bitcoin-contract-list/bitcoin-contract-list.tsx

```diff
@@ -16,10 +16,16 @@ export function BitcoinContractList() {
   const { getAllSignedBitcoinContracts } = useBitcoinContracts();
   const [bitcoinContracts, setBitcoinContracts] = useState<BitcoinContractListItem[]>([]);
   const [isLoading, setLoading] = useState(true);
+  const [isError, setError] = useState(false);
 
   useOnMount(() => {
     const fetchAndFormatBitcoinContracts = async () => {
       const fetchedBitcoinContracts = await getAllSignedBitcoinContracts();
+      if (!fetchedBitcoinContracts) {
+        setError(true);
+        setLoading(false);
+        return;
+      }
```

Comment:
If the getAllSignedBitcoinContracts function was unable to retrieve data, the page will display an error message indicating that the functionality is currently unavailable.